docs/dependencies.rst

```diff
@@ -314,7 +314,7 @@ If you want to add a dependency to the dev environment only, you can also:
 Integration with conda
 ======================
 
-conda_ is an open-source package manager very popular in the Python
+Conda_ is an open-source package manager very popular in the Python
```

Comment:
Aren't those links case-sensitive?

Reply:
Good question. Not on the GitHub preview. But that's probably not the right place to be testing it.

Reply:
I built locally, seems to work.

![image](https://user-images.githubusercontent.com/15216687/120074613-f18ed400-c09d-11eb-9d02-d46786a2a0a2.png)

Reply:
Cool, thanks for checking!